### PR TITLE
Add Arch Linux support

### PR DESCRIPTION
**Pull request by GoliathLabs**
Greetings,
I've readded the Arch Linux support cause I and my school (with about 1500 pupils) uses openvpn daily. I plan to use this script in the next time to refactor the openvpn setup properly. I'd help you to test changes made to the setup with archlinux (as my servers are powered by archlinux only).
I'd be glad if the changes could be merged again. (further testing needed which I'll do this evening).

Cheers,
Felix

### CODE REVIEW

**Comment by angristan** [comment]
Thanks, I was planning on adding it back.

File: openvpn-install.sh [hunk 1]
```diff
@@ -525,6 +554,29 @@ function installOpenVPN () {
...
+		echo ""
+		echo "As you're using ArchLinux, I need to update the packages on your system to install those I need."
+		echo "Not doing that could cause problems between dependencies, or missing files in repositories."
+		echo ""
+		echo "Continuing will update your installed packages and install needed ones."
+		until [[ $CONTINUE == "y" || $CONTINUE == "n" ]]; do
+			read -rp "Continue ? [y/n]: " -e -i y CONTINUE
+		done
+		if [[ "$CONTINUE" = "n" ]]; then
+			echo "Ok, bye !"
```
Comment (mikroskeem) [inline openvpn-install.sh]: Could this be more professional?

Comment (GoliathLabs) [reply openvpn-install.sh]: I've just re-added what was there before.

**Comment by mikroskeem** [comment]
I did some reordering (and squashing) of commits to make history look nicer.

**Pull request merged**